[docs] Add 'Integrations' section (#712)
gitpod-io/website: src/docs/bitbucket-integration.md

# Bitbucket Integration

> Gitpod comes with a [Bitbucket](https://bitbucket.org/) integration that allows you to start workspace from a Bitbucket project.

Author: ```suggestion
Gitpod comes with a [Bitbucket](https://bitbucket.org/) integration that allows you to start Gitpod workspaces from any Bitbucket project.
```

## Starting Workspaces

To start a workspace from a Bitbucket project, prefix the Bitbucket URL with `gitpod.io/#` as described in [Getting Started](/docs/getting-started/).

## Enabling Prebuilds

> To enable [prebuilt workspaces](/docs/prebuilds/) for your Bitbucket project you need to grant `install webhooks` in [Access Control](/access-control/). That allows Gitpod to install a webhook. The webhook creates a prebuild for every push event.

Author: ```suggestion
To enable [prebuilt workspaces](/docs/prebuilds/) for your Bitbucket project you need to grant `install webhooks` in Gitpod's [Access Control](/access-control/) page. This allows Gitpod to install a webhook. The webhook creates a new prebuild for every push event.
```

> The webhook will be installed when the first prebuild is triggered manually. For this, prefix a Bitbucket project URL with `gitpod.io/#prebuild/` as described on the [prebuilds page](/docs/prebuilds/#on-gitlab-and-bitbucket).

Author: ```suggestion
The webhook will be automatically installed when you trigger the first prebuild manually. For this, prefix your Bitbucket project URL with `gitpod.io/#prebuild/` as described in the [prebuilds page](/docs/prebuilds/#on-gitlab-and-bitbucket).
```
